build-artifacts/project-template-gradle/src/main/java/com/tns/NativeScriptApplication.java

```diff
@@ -12,7 +12,10 @@ public NativeScriptApplication() {
 
     public void onCreate() {
 		super.onCreate();
-		new RuntimeHelper(this).initRuntime();
+		com.tns.Runtime runtime = new RuntimeHelper(this).initRuntime();
```

Comment:
can we do this like this 
com.tns.Runtime runtime = new RuntimeHelper(this);
runtime.initRuntime()

that line look very strange to me..

Other way I would do it is to make this a static method so

com.tns.Runtime runtime = RuntimeHelper.initRuntime(this);